src/models/subscription.js

```diff
@@ -0,0 +1,33 @@
+export default (sequelize, DataTypes) => {
+  const Subscription = sequelize.define('subscription', {
+    id: {
+      type: DataTypes.UUID,
+      primaryKey: true,
+      defaultValue: DataTypes.UUIDV4
+    },
+    userId: {
```

Comment:
No need to define these fields in the model, they are added automatically by Sequelize by the associations you defined.